### ISSUE TITLE
[BUG] audio being too low.

### ISSUE THREAD

**Issue by DeepakTCO** [anilbeesetti/nextplayer#1081]
**Describe the bug**
The volume of media is too low on both speakers and my tws even if i turn on the volume boost option, i don't know why. On the other hand when i play the media in vlc even 90% of it's volume (and it has 200% too)is more than 200% of next player. It's been annoying and keeps stopping me from using my favourite video player.

**To Reproduce**
Just play anything on next player and vlc and compare the volume 

**Expected behavior**
Should be loud enough 

**Device info (please complete the following information):**
 - Device: poco f4 running a custom rom
 - Android version: 14
 - App version: 0.12.2

**Comment by drogga** [comment]
Yeah, it seems/probably VLC has Audio preAMP / Normalizer by default, or the codecs it uses do...
Years ago I has to touch AC3's settings in Win to boost all movie's Audios by default, never had cracking sounds or ever being boosted/gain-ed too much.

**Comment by drogga** [comment]
Yeah, it seems/probably VLC has Audio preAMP / Normalizer by default, or the codecs it uses do, @anilbeesetti should probably take a look sooner than later...

_Years ago I had to touch AC3's settings in Win to boost all movies Audios by default, never had cracking sounds or ever being boosted/gain-ed too much._

**Comment by DeepakTCO** [comment]
Any update to this? Audio on tws is fine now but in speaker even 200 % is just so much low

**Comment by drogga** [comment]
Are you sure those percentages aren't just in player, while the system **media** isn't to 100% max, making it something like 200% in NextPalyer of the 30% total system volume and any particular video with any particular audio codec this happens on/with ?
NP has options for using the system vol. slider, instead of the player's, I believe it also has a preference regarding headsets, but I'm not sure, as I don't have it on hand rn.
Can there be a problem with the custom rom you are running in regards to the speakers and the audio output mix source this app uses (not of the media, but a system thing) ?, for example I had issues back when I used NewPipe (I don't for years), because it uses a mix source (IDR the technical term) that bypassed my system-wide EQ app and the audios got super loud for no reason and then back to normal tru the EQ (super annoying and unpleasant), it probably still does this.

Usually when maxing out the volume in the player and dragging the notifications panel from the StatusBar while the video is playing and pressing the volume buttons should bring the system slider, which should also be reflecting the volume in the app from 0 to 100%, above is a boost/gain and won't be shown in the system vol. pop-up.

What's the situation with **Just (Video) Player** ?

**Comment by DeepakTCO** [comment]
You are right sometimes my system volume isn't set to max and that's when next player's audio is too low  i just assumed it would sync automatically with system volume. Now it's fine. Also i played the video with just player too now both has same amount of volume and it's loud enough. What i am interested in knowing is how vlc's volume is like 3X of the orginal video 😭. Like it's so loudddddd. I wish i could uave that in next player.
